src/libsyntax/attr/mod.rs

```diff
@@ -435,6 +435,30 @@ pub fn find_by_name(attrs: &[Attribute], name: Symbol) -> Option<&Attribute> {
     attrs.iter().find(|attr| attr.check_name(name))
 }
 
+pub fn allow_internal_unstable<'a>(
```

Comment:
```suggestion
pub fn allow_internal_unstable_list<'a>(
```
(suggests an action performed as currently named)